lib/shared-sku-app/src/AppConfig/AppConfig.spec.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { render, cleanup, wait, fireEvent } from '@testing-library/react';
+
+import { AppExtensionSDK } from 'contentful-ui-extensions-sdk';
+
+import AppConfig from './AppConfig';
+import { definitions } from './parameters.spec';
+
+const contentTypes = [
+  {
+    sys: { id: 'ct1' },
+    name: 'CT no 1',
+    fields: [{ id: 'obj', name: 'Some object', type: 'Object' }]
```

Comment:
Looks like this file still tests DAM scenario.

Reply:
has been updated, file was still wip